src/Microsoft.AspNet.Mvc.Razor/Compilation/CompilerCache.cs

```diff
@@ -54,9 +55,10 @@ public CompilerCache(IFileProvider fileProvider)
                 throw new ArgumentNullException(nameof(precompiledViews));
             }
 
+            var expirationTokens = new IChangeToken[0];
```

Comment:
Might be nice to add a comment calling out that we're explicitly calling out precompiled views as non-expiring